packages/cli/src/base.ts

```diff
@@ -9,15 +9,17 @@ import { Command, Flags } from '@oclif/core'
 import chalk from 'chalk'
 import net from 'net'
 import Web3 from 'web3'
-import { getGasCurrency, getNodeUrl } from './utils/config'
-import { enumEntriesDupWithLowercase, requireNodeIsSynced } from './utils/helpers'
-export const gasOptions = {
-  auto: 'auto',
-  Auto: 'auto',
-  ...enumEntriesDupWithLowercase(Object.entries(Token)),
-  ...enumEntriesDupWithLowercase(Object.entries(StableToken)),
-}
-
+import { getNodeUrl } from './utils/config'
+import { requireNodeIsSynced } from './utils/helpers'
+/**
+ *
+ * I defined a `getGasOptions` function in `helpers.ts` that makes a contract call to fetch the
+ * whitelisted address, but I need to consider that the list of gasOptions is different per
+ * network (alfajores, mainnet, baklava), because:
+ * 1. the fee currency addresses are different per network,
+ * 2. the whitelist is different, alfajores has more addresses for internal testing purposes.
+ *
```

Comment:
this sounds like a TODO. 

what im reading between the lines is that you want to check that the changing of network resets any cached whitelist

Reply:
this now seems like a stale comment